html/host.html

```diff
@@ -6,63 +6,105 @@
+                <option>[2] Memory</option>
+                <option>[2] Oil Cartel Alternate</option>
+                <option>[2] Public Good</option>
+                <option>[1] BeeSmart</option>
+                <option>[1] Bug Hunters Adaptations</option>
+                <option>[1] Bug Hunters Competition</option>
+                <option>[1] Dice Stalagmite</option>
+                <option>[1] Fish Spotters</option>
+                <option>[1] Gridlock</option>
+                <option>[1] Guppy Spots</option>
+                <option>[1] PANDA BEAR</option>
+                <option>[1] Polling</option>
+                <option>[1] Polling Advanced</option>
+                <option>[1] Prisoners Dilemma</option>
+
+                <option>[a] Oil Cartel</option> <!-- requires narrowcast plots -->
+
+                <option>[0] Restaurants</option> <!-- makes no distinction between AI restaurants and player-owned ones -->
+
+                <option>[0] Bird Breeders</option> <!-- requires spectator clients -->
+                <option>[0] Gridlock Alternate</option> <!-- requires spectator clients -->
+
+                <option>[0] Client Overrides Example</option> <!-- requires view overrides -->
+                <option>[0] Critter Designers</option> <!-- requires view overrides -->
+                <option>[0] Current in a Wire</option> <!-- requires view overrides -->
+                <option>[0] MTG 1 Equal Opportunities</option> <!-- requires view overrides -->
+                <option>[0] MTG 2 Random Assignment</option> <!-- requires view overrides -->
+                <option>[0] MTG 3 Feedback Loop</option> <!-- requires view overrides -->
+                <option>[0] Sampler</option> <!-- requires view overrides -->
+              </select>
+            </div>
+            <p id="activity-text-short" class="activity-text-short">
+              This is a placeholder for a condensed activity description.
+            </p>
+            <input type="button" id="read-more-button" class="read-more-button" value="Read More"></input>
+          </div>
+
+          <div class="image-wrapper">
+            <img
+              id="preview-image"
+              class="preview-image"
+              src=" http://ccl.northwestern.edu/netlogo/models/models/Curricular%20Models/epiDEM/epiDEM%20Basic.png"
+              alt="Activity Image"
+            >
+          </div>
+        </div>
+
+        <div class="form-input-wrapper">
+          <div>
+            <label class="form-label">Session Name
+              <input class="form-input" name="sessionName" type="text" pattern="\w.*" required>
+            </label>
+          </div>
+          <div class="center-input">
+            <label class="form-label">Password
+              <input class="form-input" name="password" type="text" required>
+            </label>
+          </div>
+          <input class="button-general launch-button" type="submit" value="Launch">
```

Comment:
See if you can't vertically center this button with the `input` elements.  I understand that that means it will extend below their lowest point.  My intuition is that this will look better.

Reply:
Tried to approximate this as best I could! Think it looks pretty similar tbh, but not 100% sure.